server/src/main/java/game/GameManager.java

```diff
@@ -36,7 +36,7 @@ public String createGame(Player host, int maxPlayers, String gameName) {
         this.notPlayingGameList.put(game.getGameId(), game);
         SocketServer.getInstance().addGame(game.getGameId());
         clientProxy.updateGameList(new ArrayList<>(notPlayingGameList.values()));
-
+        joinGame(game.getGameId(), host);
```

Comment:
Is there a benefit to moving joinGame here rather then in the serverFacade itself?

Reply:
Yeah. The server was throwing an Illegal Access Exception and it was fixed by moving it there.